src/app/games/epaminondas/epaminondas.component.ts

```diff
@@ -136,7 +136,8 @@ export class EpaminondasComponent extends AbstractGameComponent<EpaminondasMove,
         return extensions;
     }
     private getPhalanxValidExtensions(PLAYER: number): Coord[] {
-        let direction: Direction = Direction.factory.fromMove(this.firstPiece, this.lastPiece);
+        // TODO FOR REVIEW: is the .get() always safe? If not, maybe cancelMove?
+        let direction: Direction = Direction.factory.fromMove(this.firstPiece, this.lastPiece).get();
```

Comment:
TODO's answer: I'd say yes cause it has to have been tested while choosing this.lastPiece, and my politic on private methods in component is TDD over [over]defensive programming.
So yep, rather always safe.

Reply:
ok, nice!